src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,34 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_info(self, type='object'):
```

Comment:
Most of the Testers I know, only use Selenium2Library, they don't need to get into Selenium (or know about it). As I said before, for example, we don't control the browser version, and we are getting the current information or capabilities.